src/extension.ts

```diff
@@ -54,6 +55,7 @@ function InstantMarkdown() {
         return '';
       }
     });
+    md.use(mdnh);

```

Comment:
Thank for the PR. I think you can just chain the usage with the other extensions in the next line of code.